### PR TITLE
Add qarrayrun from pypi

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge/help-r This is ready for review. I used the pypi skeleton, not sure if that's following conda-forge best-practices for python though, so please advise if not.

File: recipes/qarrayrun/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "qarrayrun" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "80441f5a0cf27e3ba8805ced549f27c176f43d356e6d783c33ef643585a106c5"
```
Comment (marcelotrevisani) [inline recipes/qarrayrun/meta.yaml]: Could you please remove the quotes from the ``sha256``?
```suggestion
  sha256: 80441f5a0cf27e3ba8805ced549f27c176f43d356e6d783c33ef643585a106c5
```

Related to https://github.com/conda/conda-build/pull/3720

File: recipes/qarrayrun/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  entry_points:
+    - qarrayrun = qarrayrun.cli:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/qarrayrun/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

The code on ``conda-build`` was updated. However, I think there is not a new release with those modifications

Comment (dpryan79) [reply recipes/qarrayrun/meta.yaml]: Done

File: recipes/qarrayrun/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/qarrayrun/meta.yaml]: Sure, done

**Pull request merged**